lib/rubycritic/analysers/smells/flog.rb

```diff
@@ -1,10 +1,12 @@
 require "rubycritic/analysers/helpers/flog"
 require "rubycritic/core/smell"
+require "rubycritic/colorize"
 
 module Rubycritic
   module Analyser
 
     class FlogSmells
+      include Colorize
```

Comment:
How about creating something like a BaseAnalyzer that includes Colorize and then have all other analyzers inherit from it?